### PR TITLE
[KOGITO-4267] Kafka topic based on message name

### PR DESCRIPTION
**Pull request by fjtirado**
Adding example using multi kafka addon for Quarkus and Springboot

Merge with https://github.com/kiegroup/kogito-runtimes/pull/1235

### CODE REVIEW

File: process-kafka-multi-quarkus/mvnw [hunk 1]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/mvnw]: @fjtirado we can probably skip adding this to git, there is one already in the root level

File: process-kafka-multi-quarkus/mvnw.cmd [hunk 2]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/mvnw.cmd]: same here

File: process-kafka-multi-quarkus/operator/process-kafka-quickstart-quarkus.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+metadata:
+  name: process-kafka-quickstart-quarkus
+spec:
+  type: RemoteSource
+  #env:
+  # env can be used to set variables during build
+  #- name: MY_CUSTOM_ENV
+  #  value: "my value"
+  gitSource:
+    contextDir: process-kafka-quickstart-quarkus
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/operator/process-kafka-quickstart-quarkus.yaml]:
```suggestion
    contextDir: process-kafka-multi-quickstart-quarkus
```

File: process-kafka-multi-quarkus/operator/process-kafka-quickstart-quarkus.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    uri: 'https://github.com/kiegroup/kogito-examples'
+  # set your maven nexus repository to speed up the build time
+  #mavenMirrorURL:
+---
+apiVersion: app.kiegroup.org/v1beta1
+kind: KogitoRuntime
+metadata:
+  name: process-kafka-quickstart-quarkus
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/operator/process-kafka-quickstart-quarkus.yaml]:
```suggestion
  name: process-kafka-multi-quickstart-quarkus
```

File: process-kafka-multi-quarkus/pom.xml [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-examples</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>process-kafka-multi-quarkus</artifactId>
+  <name>Kogito Example :: Process with Kafka and Quarkus</name>
+  <description>Kogito with Kafka - Quarkus</description>
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/pom.xml]: can we adjust this so it is different from the other example?

File: process-kafka-multi-quarkus/src/main/docker/Dockerfile.jvm [hunk 6]
```diff
@@ -0,0 +1,26 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the docker image run:
+#
+# mvn package
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/process-kafka-quickstart-quarkus-jvm .
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/src/main/docker/Dockerfile.jvm]:
```suggestion
# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/process-kafka-multi-quarkus-jvm .
```

File: process-kafka-multi-quarkus/src/main/docker/Dockerfile.jvm [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/process-kafka-quickstart-quarkus-jvm
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/src/main/docker/Dockerfile.jvm]:
```suggestion
# docker run -i --rm -p 8080:8080 quarkus/process-kafka-multi-quarkus-jvm
```

File: process-kafka-multi-quarkus/src/main/docker/Dockerfile.native [hunk 8]
```diff
@@ -0,0 +1,22 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in native (no JVM) mode
+#
+# Before building the docker image run:
+#
+# mvn package -Pnative -Dnative-image.docker-build=true
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.native -t quarkus/kafka-quickstart .
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/src/main/docker/Dockerfile.native]:
```suggestion
# docker build -f src/main/docker/Dockerfile.native -t quarkus/process-kafka-multi-quarkus .
```

File: process-kafka-multi-quarkus/src/main/docker/Dockerfile.native [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/kafka-quickstart
```
Comment (cristianonicolai) [inline process-kafka-multi-quarkus/src/main/docker/Dockerfile.native]:
```suggestion
# docker run -i --rm -p 8080:8080 quarkus/process-kafka-multi-quarkus
```

**Comment by cristianonicolai** [comment]
@fjtirado could we also add a spring based example?

**Comment by fjtirado** [comment]
> @fjtirado could we also add a spring based example?

Hmmm, not sure about that. I was following the idea of @ricardozanini of making Quarkus first class citizen. Do we really need the springboot counterpart for all examples? This seems a good place to start saving work related with Springboot

File: process-kafka-multi-quarkus/mvnw (hunk 1 above)
Comment (fjtirado) [reply process-kafka-multi-quarkus/mvnw]: done

**Comment by fjtirado** [comment]
Adding springboot example

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**